docs/source/algorithms/SaveGDA-v1.rst

```diff
@@ -113,7 +113,7 @@ Output:
 
 .. testoutput:: SaveGDA
 
-    BANK 4 4246  1062 RALF  27388  96  27388 0.001 ALT
+    BANK 1 4246  1062 RALF  27388  96  27388 0.001 ALT
```

Comment:
Why did the output of this algorithm change?

Reply:
In the old GEM scripts bank numbers went from high to low in the GDA and MAUD files (and they need to be in the same order in both files), so when we could only generate the GDA from Mantid we kept them in descending order so GEM could use the GDA with their old MAUD file. GEM prefer them to be ascending, so now that we can generate both files from Mantid, which happened in this PR, the bank numbers can be reversed in the GDA

Reply:
Ok. Sounds good.

Reply:
But does it mean we're no longer backwards compatible?

Reply:
It means that a GDA made with our scripts won't work with a MAUD made with theirs, and vice versa. Previously both files were made with scientist-written scripts that made no mention of Mantid, so it's not a case of not being backwards compatible with ourselves. The scientists also understand that this is the case and are happy with it